### DISCUSSION TITLE
How to configure a Keycloak trustore in Keycloak Docker image/Kubernetes

### DISCUSSION THREAD

**Discussion by blevine** [keycloak/keycloak#24609] [Q&A]
I build my own Keycloak Docker image based on the jboss/keycloak Docker image version 12.0.4. The custom image includes themes and a few custom plugins. This Docker image is deployed to a Kubernetes cluster using a Helm chart. I'm testing out a brokered IdP configuration against Auth0 and I'm almost there. The last thing I need is to configure a truststore with Auth0's cert.

I'm thinking the best way to do this would be to include the trust store when I build my Docker image, but I'm not quite sure how to go about this. Any suggestions?

Note: upgrading to later version of Keycloak is not an option at this time.

**Comment by vmuzikar** [comment]
Please note Keycloak 12 is very old version, unsupported for a long time now and exposed to various security vulnerabilities. Please consider upgrading.
